c/rust/Makefile

```diff
@@ -14,6 +16,16 @@ build:
 	    ;; \
 	esac
 
+libvirtblocks_c_rust.la: ../libtool.la.in rustbuild
+	mkdir -p .libs/
+	ln -sf ../target/debug/$(@:.la=.a) .libs/$(@:.la=.a)
+	sed \
+		-e \
+			's/@LIBNAME@/$(@:.la=)/g' \
+		-e \
+			"s/^inherited_linker_flags=.*/inherited_linker_flags='-pthread -lm -ldl'/" \
+		<$< >$@
```

Comment:
ewww to that indentation! Please change it to something more reasonable like

```
sed \
  -e 's/.../.../g' \
  -e 's/.../.../g' \
  <$< >$@
```

Also ewww to replacing linker flags that way: just have

```
inherited_linker_flags='@LDFLAGS@'
```

in the template and then replace it appropriately in the Rust and Go `Makefile`s.